src/components/layout/Navbar.tsx

```diff
@@ -42,29 +60,50 @@ const navItems: NavItem[] = [
     label: 'Verification',
     url: '/verification',
   },
+  {
+    label: 'Settings',
+    url: '/settings',
+  },
   {
     label: 'SearchSECO',
     pages: [
-      { label: 'Query', url: '/query' },
-      { label: 'Mining', url: '/mining' },
+      {
+        label: 'Query',
+        url: '/query',
+        icon: HiOutlineDocumentMagnifyingGlass,
+        description: 'Query the SearchSECO database using your $SECOINS',
+      },
+      {
+        label: 'Mining',
+        url: '/mining',
+        icon: HiOutlineTerminal,
+        description: 'Claim your mining rewards in $SECOINS or $SECOREP',
```

Comment:
Same here as above comment for the SECOIN and SECOREP, use `TOKENS.secoin.symbol` and `TOKENS.rep.symbol`